### DISCUSSION TITLE
Bringing back PWA ?

### DISCUSSION THREAD

**Discussion by gunir** [Floorp-Projects/Floorp#438] [Feature Request]
Firefox killed PWA for Firefox after version 84, I believe there's still old source code of this component, would be great to bring it back!

PWA is very useful in some cases:
- Isolate Floorp's cache, indexedDB to prevent slowness (if cache is too big or indexedDB is too big)
- Isolate huge web service from Floorp to optimize resource usage
- I know that PWAforFirefox exists, but it doesn't look very comfortable because it requires users to install PWA Launcher, which is uncomfortable

**Comment by creeper-0910** [comment]
https://github.com/Floorp-Projects/Floorp/discussions/411
duplication.

**Comment by Ryosuke-Asano** [comment]
https://github.com/Floorp-Projects/Floorp/discussions/411#discussioncomment-7800301
